app/lib/helpers/helpers.ts

```
@@ -0,0 +1,288 @@
+    );
+  }
+
+  if (txt.length > wrapAfter) {
+    return txt.slice(0, wrapAfter) + '...';
+  }
+
+  return txt;
+};
+
+export const titleCase = (txt: string) => {
+  if (typeof txt != 'string') {
```

Biome (1.9.4)

[error] Use !== instead of !=. != is only allowed when comparing against `null`

!= is only allowed when comparing against null

Using != may be unsafe if you are relying on type coercion
Unsafe fix: Use !==


(lint/suspicious/noDoubleEquals)